src/components/shared/Entity/Details/useRelatedEntities.tsx

```diff
@@ -4,27 +4,38 @@ import { useIntl } from 'react-intl';
 
 import RelatedEntities from 'src/components/shared/Entity/Details/RelatedEntities';
 import { useEntityType } from 'src/context/EntityContext';
+import { ENTITY_SETTINGS } from 'src/settings/entity';
 import { useEntityRelationshipStore } from 'src/stores/EntityRelationships/Store';
 
 function useRelatedEntities() {
     const intl = useIntl();
     const entityType = useEntityType();
 
-    const [relatedCaptures, relatedMaterializations, hydrationError, hydrated] =
-        useEntityRelationshipStore((state) => [
-            state.captures,
-            state.materializations,
-            state.hydrationError,
-            state.hydrated,
-        ]);
+    const [
+        relatedCaptures,
+        relatedMaterializations,
+        relatedCollections,
+        hydrationError,
+        hydrated,
+    ] = useEntityRelationshipStore((state) => [
+        state.captures,
+        state.materializations,
+        state.collections,
+        state.hydrationError,
+        state.hydrated,
+    ]);
 
     return useMemo(() => {
         const response = [];
 
-        if (entityType === 'collection') {
+        if (
+            ENTITY_SETTINGS[entityType].details.relatedEntitiesContentIds
+                .writtenBy
+        ) {
```

Comment:
We need to store these keys off to use anyway - so using them as a flag here. I _somewhat_ am okay with this and _somewhat_ not. I think time will tell how this feels.